internal/api/handler/explore.go

```diff
@@ -19,5 +22,41 @@ func NewExploreHandler(exploreRepo repo.ExploreRepository) *exploreHandler {
 }
 
 func (e *exploreHandler) HandleExplore(w http.ResponseWriter, r *http.Request) {
-	w.WriteHeader(http.StatusOK)
+	// Normalize path param
```

Comment:
please be more specific on what normalize means,

Reply:
Expanded comment